dev/benchmarks/macrobenchmarks/macos/Runner.xcodeproj/project.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is a mistake that I didn't notice was added to the template. Someone just caught this and sent a PR recently that fixed the template, but this file should be removed here.

Reply:
Removed